addons/sale/models/account_move.py

```diff
@@ -46,6 +46,22 @@ def _compute_origin_so_count(self):
         for move in self:
             move.sale_order_count = len(move.line_ids.sale_line_ids.order_id)
 
+    @api.depends('partner_id')
+    def _compute_invoice_default_sale_person(self):
+        super()._compute_invoice_default_sale_person()
```

Comment:
With this, we're computing twice. Shouldn't we filter out the moves we compute below, and do the super call on the remaining onces ? Unless the super call is computing other things as well ? :thinking: